src/components/PostList/PostList.jsx

```diff
@@ -1 +1,9 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings